examples/confirm/enquirer-option-default.js

```diff
@@ -9,10 +9,12 @@ prompt({
   initial: 'j',
   default: '(J/n)',
   isTrue(input) {
-    return input.toLowerCase() === 'j';
+    const isString = typeof input === 'string';
+    return isString ? input.toLowerCase() === 'j' : false;
```

Comment:
I think all we need here is to do `isTrue(input = '') {`

Reply:
I tried `isTrue(input = '')`, but it is giving the same error.
This is because `input` also gets Boolean values `true` and `false` which overwrites default `''`.
We want to ensure that `toLowerCase` function is only invoked on string values.

Reply:
Ah... I see. In that case, we want to make sure Booleans are handled correctly too:

```js
  isTrue(input) {
    return typeof input === 'boolean'
      ? input
      : input.toLowerCase() === 'j';
  },
  isFalse(input) {
    return typeof input === 'boolean'
      ? !input
      : input.toLowerCase() === 'n';
  }
```

I think that should be enough to demonstrate this example.

Reply:
I agree.